consensus/types/src/chain_spec.rs

```diff
@@ -513,6 +513,10 @@ pub struct YamlConfig {
     inactivity_penalty_quotient: u64,
     #[serde(with = "serde_utils::quoted_u64")]
     min_slashing_penalty_quotient: u64,
+    // Proportional slashing multiplier defaults to 3 for compatibility with Altona and Medalla.
+    #[serde(default = "default_proportional_slashing_multiplier")]
```

Comment:
I added this, but I think it's actually preferable to just add the `PROPORTIONAL_SLASHING_MULTIPLIER` to the old testnet configs